### ISSUE TITLE
Merge Master before launch

### ISSUE THREAD

**Issue by dougkdev** [FreeUKGen/FreeCENMigration#206]
It has been a long time since master merged into freecen_parsing. This one will be painful, requiring quite a bit of manual resolution of code conflicts and manual review for conflicts that may not be auto-detectable. It will also require a FC2 database rebuild and re-copy to production afterwards, but I highly recommend doing it before public launch.

**Comment by dougkdev** [comment]
This goes hand-in-hand with #207. Merge master, do #206 (rebuild db on vervet), test both together on vervet, deploy db to production (#207), deploy merged codebase to production. I recommend that Doug and Ben do the merge to resolve any existing conflicts, then Doug help new developer rebuild the fc2 mongo database, then Ben help new developer do the copy to production.

**Comment by richpomfret** [comment]
Rebuild needed & tested post UCF tasks being completed.

**Comment by benwbrum** [comment]
The latest UCF fix for infix-* notation needs to be merged before we can deploy.
